setup.cfg

```diff
@@ -1,5 +1,5 @@
 [tool:pytest]
```

Comment:
Yes, I think so! Will double check that no built assets are included in the repo!